tests/Makefile

```diff
@@ -15,6 +15,9 @@ ODIR ?= obj
 
 LDFLAGS += -L.
 
+# Enabling benchmarks
+DEFINES += -DCATCH_CONFIG_ENABLE_BENCHMARKING
```

Comment:
`CATCH_CONFIG_ENABLE_BENCHMARKING` should be also added to VS project (I've did this for you).